src/core/qgssettings.cpp

```diff
@@ -177,12 +177,13 @@ QString QgsSettings::fileName() const
 
 void QgsSettings::sync()
 {
-  return mUserSettings->sync( );
+  return mUserSettings->sync();
 }
 
 void QgsSettings::remove( const QString &key )
 {
-  mGlobalSettings->remove( key );
```

Comment:
@jef-n please see https://github.com/qgis/QGIS/pull/4160/files#r103233230 (I'll fix it when back from vacation later this week, but feel free to do it earlier if you will). See also http://hub.qgis.org/issues/16233